src/api/todos.ts

```diff
@@ -1,8 +1,33 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
+export const getTodos = (userId?: number) => {
```

Comment:
Are you sure that userId is not required here? How can you get todos if there will be no user?
```suggestion
export const getTodos = (userId: number) => {
```

Reply:
About user. In the code in other files, whitch were written before user can be null. 
![image](https://user-images.githubusercontent.com/98876779/220150400-d71513c6-feb0-4d78-8f79-8687c8c6ca2b.png)